src/molecules/gv-card-full/gv-card-full.js

```diff
@@ -134,13 +134,17 @@ export class GvCardFull extends ItemResource(LitElement) {
         .description {
           padding: 0 16px;
           margin: 6px 0;
-          flex: 1;
-          flex-grow: 1;
           max-height: 150px;
+          flex: 1;
           overflow: hidden;
           text-overflow: ellipsis;
         }
 
+        .description * {
```

Comment:
This will apply to all sub-elements, won't it?

Reply:
Yes but just for text and display, If someone adds something that renders to a block the UI looks really 💥